### PR TITLE
finishes makeChange()

### PR DESCRIPTION
**Pull request by mistakeets**
makeChange() function will take given input and return the amount of quarters, dimes, nickels, and pennies it takes to make the correct change.

### CODE REVIEW

File: src/makeChange.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+  var dimeCount = Math.floor((changeStart%25)/10)
+  var nickelCount = Math.floor(((changeStart%25)%10)/5)
+  var pennyCount = Math.floor(((changeStart%25)%10)%5)
+
+    return {
+      quarters: quarterCount,
+      dimes: dimeCount,
+      nickels: nickelCount,
+      pennies: pennyCount
+    }
```
Comment (devonwesley) [inline src/makeChange.js:13]: Nice and concise function. Goodjob.

**Comment by devonwesley** [comment]
Nice go ahead and merge it.

File: src/makeChange.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
 export default function makeChange({price, amountGiven}) {
-  // your code here
+  var changeStart = amountGiven - price
+  var quarterCount = Math.floor(changeStart/25)
+  var dimeCount = Math.floor((changeStart%25)/10)
+  var nickelCount = Math.floor(((changeStart%25)%10)/5)
+  var pennyCount = Math.floor(((changeStart%25)%10)%5)
```
Comment (jaredatron) [inline src/makeChange.js:6]: For what it's worth, I'd have written it like this:
```js
const change   = amountGiven - price
const quarters = Math.floor(change / 25)
const dimes    = Math.floor((change % 25) / 10)
const nickels  = Math.floor(((change % 25) % 10) / 5)
const pennies  = Math.floor(((change % 25) % 10) % 5)

return {
  quarters,
  dimes,
  nickels,
  pennies,
}
```

Comment (mistakeets) [reply src/makeChange.js:6]: makes sense thanks for the feedback 👍

**Pull request merged**